Jenkinsfile

```diff
@@ -38,7 +38,7 @@ def transformIntoStep(arch, wspwd, container) {
                                                          -v ~/.ccache:/workdir/.ccache \
                                                          \$VOLUMES \
                                                          --name "${env.BUILD_TAG}-${arch}" \
-                                                         ${container} -c \
+                                                         ${container} bash -c \
```

Comment:
Why is this required? Should it be able to execute the shell script directly which will open a bash anyway?

Reply:
nope, the image does not have default command set, so you have to explicitly call ist,

without you get this:
```container_linux.go:247: starting container process caused "exec: \"-c\": executable file not found in $PATH"```

Reply:
What is the `-c` doing exactly? Would removing this fix it? --> `exec ...foo.sh` should work as the script is executable and has a bash shebang

Reply:
yep you're right, should work too

Reply:
ah I've found a reason to still include it. you get a different parameter unescaping if you do not call it "inside" the container, but let the docker assign the parameters.

(i.e if you manually try to add ">&2 | less" it doesnt work if you do not run it via "bash -c"

Reply:
ok then